src/main/java/net/sf/jsqlparser/statement/select/PlainSelect.java

```diff
@@ -407,25 +407,29 @@ public static String getStringList(List<?> list) {
      * @return comma separated list of the elements in the list
      */
     public static String getStringList(List<?> list, boolean useComma, boolean useBrackets) {
-        String ans = "";
+        StringBuilder ans=new StringBuilder();
+//        String ans = "";
         String comma = ",";
         if (!useComma) {
             comma = "";
         }
         if (list != null) {
             if (useBrackets) {
-                ans += "(";
+                ans.append("(");
+//                ans += "(";
             }
 
             for (int i = 0; i < list.size(); i++) {
-                ans += "" + list.get(i) + ((i < list.size() - 1) ? comma + " " : "");
+                ans.append(list.get(i)).append(((i < list.size() - 1) ? comma + " " : ""));
```

Comment:
To construct valid comma lists the constructs are nearly always weird :). You could use a commalist generator method within Plainselect. Nevertheless it was not your statement but the older one I am refering to.